packages/client/lib/sync/sync.ts

```diff
@@ -174,7 +174,7 @@ export abstract class Synchronizer {
         this.clearFetcher()
         resolve(true)
         const heightStr = isTruthy(height) ? ` height=${height}` : ''
-        this.config.logger.debug(
+        this.config.logger.info(
           `Finishing up sync with the current fetcher${heightStr}
```

Comment:
```suggestion
          `Finishing up sync with the current fetcher ${heightStr}
```
There's a weird newline in this log (probably from `prettier`).  I can't add it in this comment but you need to delete the newline as well that runs into line 179

Reply:
done!